addons/l10n_pk_edi/models/res_company.py

```diff
@@ -0,0 +1,13 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    l10n_pk_edi_pos_id = fields.Char("PoS ID", groups="base.group_system")
```

Comment:
better to reserve _id for many2one
```suggestion
    l10n_pk_edi_pos_ident = fields.Char("PoS ID", groups="base.group_system")
```